lib/paperclip/deprecations.rb

```diff
@@ -0,0 +1,34 @@
+require 'active_support/deprecation'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.